Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -466,6 +466,45 @@
         - RawMaterial
         - Ingot
 
+- type: entity
+  id: AmmoTechFab
+  parent: BaseLathe
+  name: ammo techfab
```

Comment:
The test is failing because you need to overwrite the `Machine` component with your `AmmoTechFabCircuitboard` container, like all techfabs do. Also if it isnt a big nuisance, can you add a video constructing/deconstructing this fab? just to confirm it all works

Reply:
machine component added, uploading video soon!